app/views/external_authors/claim.html.erb

```diff
@@ -21,21 +21,21 @@
 <div class="claim module">
   <h3 id="claim"><%= ts("Claim All Works") %></h3>
 
-  <% if logged_in? %>
+  <% if user_signed_in? %>
     <p class="message">
       <%= ts("Claim your works with your logged-in account. If you would like to claim
       these works under a different account (or create a new account to claim them with), log out and reload this
       page first.") %>
     </p>
-    <p class="actions"><%= button_to ts("Add these works to my currently-logged-in account"), complete_claim_url(:invitation_token => @invitation.token) %></p>
+    <p class="actions"><%= button_to ts("Add these works to my currently-logged-in account"), complete_claim_url(invitation_token: @invitation.token) %></p>
```

Comment:
I noticed that the options on lines 38 and 53 have been changed from `button_to` to `link_to`, while this and the one on line 51 have stayed the same. Could you explain why?